### PR TITLE
Chatbot Image size and list fixes

### PR DESCRIPTION
**Pull request by dawoodkhan82**
## Description
Fixes lmsys style regressions

Closes: #9159

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: js/chatbot/shared/ChatBot.svelte [hunk 1]
```diff
@@ -701,12 +714,6 @@
 		border: 1px solid var(--border-color-accent);
 	}
 
-	/* Lists */
-	.message-wrap :global(ol),
```
Comment (dawoodkhan82) [inline js/chatbot/shared/ChatBot.svelte:705]: @freddyaboulton Do you know why we added this?

File: js/chatbot/shared/ChatBot.svelte [hunk 2]
```diff
@@ -682,6 +682,19 @@
...
 
+	.message-wrap
+		> div
+		:not(.avatar-container)
+		div
+		:not(.image-button)
+		:global(img) {
+		border-radius: 13px;
+		margin: var(--size-2);
+		width: 400px;
```
[the remark this answers is not in the document]
Comment (dawoodkhan82) [reply js/chatbot/shared/ChatBot.svelte:693]: cool, thanks for checking!

File: js/chatbot/shared/ChatBot.svelte [hunk 3]
```diff
@@ -682,6 +682,19 @@
...
+		max-width: 30vw;
+		max-height: auto;
```
[the remark this answers is not in the document]
Comment (dawoodkhan82) [reply js/chatbot/shared/ChatBot.svelte]: oh yeah good catch, changed it

File: js/chatbot/shared/ChatBot.svelte (hunk 1 above)
Comment (freddyaboulton) [reply js/chatbot/shared/ChatBot.svelte:705]: I think this maybe existed in the chatbot before and I copied it over from my custom component when mainlined it

Comment (freddyaboulton) [reply js/chatbot/shared/ChatBot.svelte:705]: LGTM though!

**Review by freddyaboulton — APPROVED** [review]

**Pull request merged**